mobile/cmp/navigator/NavigatorModel.js

```diff
@@ -48,11 +57,18 @@ export class NavigatorModel extends HoistModel {
         return stack[stack.length - 1];
     }
 
+    /** @type boolean */
+    @computed
+    get swipeComplete() {
+        return this.swipeProgress === 1;
+    }
+
     /**
      * @param {Object[]} pages - configs for PageModels, representing all supported
      *      pages within this Navigator/App.
      * @param {boolean} [track] - True to enable activity tracking of page views (default false).
      *      Viewing of each page will be tracked with the `oncePerSession` flag, to avoid duplication.
+     * @param {boolean} [enableSwipe] - True to enable 'swipe to go back' functionality.
```

Comment:
How about `swipeToGoBack` as a slightly more verbose but also more clear name for this config?

It's going to be something that developers type once per mobile app (I assume), so I don't think the extra length is much of a penalty in practice.

I'd also vote for defaulting it to true - unless we find it causing some unexpected issues, it's such a nicely realized feature that I don't see a reason _not_ to enable it.  (If some users found it annoying for some reason, I could see this being an app-level option we allow users to turn on and off.)

Reply:
agree

Reply:
Great, will rename and change the default. Note that this change means we can close the Toolbox PR without merging: https://github.com/xh/toolbox/pull/514